src/common/_AllRetailersSlides/_AllRetailersSlides.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { RetailerCard } from "../RetailerCard/RetailerCard"
+
+export function _AllRetailersSlides({slidesData}) {
+  return (
+    <>
+      {slidesData.map(slide =>
+        <RetailerCard
```

Comment:
The same..
we use loop here and don't use `key` in RetailerCard. And why component name starts with ''_"?